app/views/lookbook/pages/show.html.erb

```diff
@@ -5,11 +5,11 @@
   @dom:update-complete.window="Lookbook.initEmbeds();">
   <div class="h-full bg-lookbook-page-bg relative">
     <% unless @error %>
-      
```

Comment:
The whitespace changes in this file seem unintentional

Reply:
Gah, my editor strips whitespace on save :( It wasn't intentional, but I don't think it would cause a problem either.

Reply:
No worries, I'll merge like this and it should get automatically removed again next time I do a release (or before!).